src/config/routes/2.1.json

```diff
@@ -144,6 +144,13 @@
     },
 
     {
+        "path": "/users(/[^/]+)*/?$",
+        "controller": "UsersController",
+        "action": "deleteAction",
```

Comment:
@heiglandreas Within the API controllers, they all follow the format [verb]Action  (getAction, postAction etc), so I believe this to be correct.